### PR TITLE
created ui for manage bookings ui issue  and made it responsive

### PR DESCRIPTION
**Pull request by MUGABA**
### The Reason for the pull request!
- This pr contains  the changes to create a manage booking interface
- In case of merging it the interface can be viewed on /bookings route!

### Usability the interface! 
- The interface has both desktop version and mobile version 
- I used static data for the demo on the card

### Future Revisits.
- This part of code will revisited to make the interface dynamic depending on the logged in user! 

### Below are some images to illustrate the screens!

Screenshot 2022-01-17 200354
eenshot/video of the problem, along with detailed console outputs where applicable

Screenshot 2022-01-17 200449

Screenshot 2022-01-17 200728

Screenshot 2022-01-17 200831
### updated interface look
Screenshot 2022-01-23 150629

Screenshot 2022-01-23 150738

### CODE REVIEW

**Comment by ezeilo-su** [comment]
Hey @MUGABA, please fill out the PR templates and link your ticket in the PR before we can proceed with code review.

**Comment by MUGABA** [comment]
> Hey @MUGABA, please fill out the PR templates and link your ticket in the PR before we can proceed with code review.

Hello! unless when there a problem when I was filling the form, though I have filled in and submitted again! For linking the issue I did it prior to creating the pr

**Review by moffatethan — CHANGES_REQUESTED** [review]
Pausing review on this

@MUGABA PR template is not filled out properly. Screenshots are provided but everything else is still the template information.

File: client/src/App.tsx [hunk 1]
```diff
@@ -28,6 +29,7 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard" component={Dashboard} />
                 <Route path="/profile/settings" component={Settings} />
+                <Route path="/booking" component={BookingsPage} />
```
Comment (moffatethan) [inline client/src/App.tsx:32]: `/my-jobs` is the page we want so it works in the navbar.

File: client/src/components/ManageBookings/BookingCalendar/BookingCalendar.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        if (upcomingBookings) {
+          for (const booking of upcomingBookings) {
+            if (isSameDay(booking.startTime, date) && booking.status != 'declined') {
+              return classes.activeTile;
+            }
+          }
+        }
+      }
+    }
+    return 'yes';
```
Comment (moffatethan) [inline client/src/components/ManageBookings/BookingCalendar/BookingCalendar.tsx:36]: Generally we should pass back true/false for truthy or falsey statements.

File: client/src/components/ManageBookings/CurrentBooking/CurrentBookingCard.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import SettingsIcon from '@mui/icons-material/Settings';
+import useStyles from './useStyles';
+
+const CurrentBookingCard = (): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Box component="div" className={classes.cardWrapper}>
+      <SettingsIcon className={classes.icon} />
+      <Typography component="div" className={classes.date}>
+        YOUR NEXT BOOKING
```
Comment (moffatethan) [inline client/src/components/ManageBookings/CurrentBooking/CurrentBookingCard.tsx:11]: Generally against using uppercase like this. Rather, we should set the CSS to transform the text to uppercase. Screen readers will read this out letter by letter which would be very annoying for a user with disabilities.

File: client/src/components/ManageBookings/CurrentBooking/useStyles.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    width: '400px',
+    height: '150px',
+    borderRadius: '5px',
+    position: 'relative',
+    outline: '0px',
+    padding: '5px',
+    margin: '0px auto',
+    backgroundColor: 'rgb(255, 255, 255)',
+    // borderWidth: '1px 1px',
+    // borderStyle: 'solid',
```
Comment (moffatethan) [inline client/src/components/ManageBookings/CurrentBooking/useStyles.ts:15]: Remove please

File: client/src/components/ManageBookings/CurrentBooking/useStyles.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  details: {
+    position: 'relative',
+    top: '-10px',
+    left: '-10px',
+    width: '100%',
+    height: '100%',
+    margin: 'auto',
+    display: 'flex',
+    flexDirection: 'row',
+    // justifyContent: 'space-between',
```
Comment (moffatethan) [inline client/src/components/ManageBookings/CurrentBooking/useStyles.ts:38]: Remove please

**Review by moffatethan — CHANGES_REQUESTED** [review]
Not sure on the UI overall, it's not the greatest. Everything appears squished and far apart. Could we work on this a bit more @MUGABA. 

Thanks!

**Comment by moffatethan** [comment]
Also - I don't like the calendar font. Could you try making it match our font? Or https://www.youtube.com/watch?v=5jRrVqRWqsM here is a video on creating a calendar in React if you want to take another stab at it, just make sure to design the UI to be like the mocks.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Lots of changes in this PR, I think we need to base the branch properly.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Please address merge conflicts

**Review by moffatethan — APPROVED** [review]

**Pull request merged**